src/lib/userStorage/UserStorageClass.js

```diff
@@ -750,7 +750,7 @@ export class UserStorage {
   // eslint-disable-next-line require-await
   async formatEvent(event: FeedEvent) {
     try {
-      return await this._formatEvent(event)
+      return this._formatEvent(event)
```

Comment:
the await is required for the catch block to happen

Reply:
reverted